packages/core/addon/components/navi-visualizations/table.js

```diff
@@ -173,6 +173,35 @@ class Table extends Component {
     }, []);
   }
 
+  /**
+   * Adds meta into rows for serverside data rollups.
+   *
+   * @param {Array<Row>} tableData - Rows of data to render
+   * @param {RequestV1} request - request in order
+   * @returns {Array<Row>} - tableData with added meta to denote rollup rows
+   */
+  _mapRollups(tableData, request) {
+    if (request?.rollup?.columns?.length <= 0 && !request?.rollup?.grandTotal) {
+      return tableData;
+    }
+    const fullMask = parseInt(new Array((request.dimensions?.length ?? 0) + 1).fill('1').join(''), 2);
+    return tableData.map(row => {
+      if (row.__rollupMask === undefined || row.__rollupMask === null) {
+        return row;
+      }
+      if (!row.__meta__) {
+        row.__meta__ = {};
+      }
+      if (row.__rollupMask !== fullMask) {
+        set(row, '__meta__.isRollup', true);
+      }
+      if (row.__rollupMask === 0) {
+        set(row, '__meta__.setisTotalRow', true);
```

Comment:
Few questions here:
- Are we using `setisTotalRow`?
- What is the diff between `totalRow` and `rollup`?
- Should this be `isTotalRow` instead of `setisTotalRow`?

Reply:
It should be `isTotalRow` it's a typo that I'm fixing.
totalRow is a grand total row. which is consistent with client side grand total row.